ai/STA/Strategy/lineup.py

```diff
@@ -0,0 +1,48 @@
+# Under MIT license, see LICENSE.txt
+from functools import partial
+
+from Util import Position, Pose
+from Util.constant import ROBOT_RADIUS
+from Util.role import Role
+from Util.role_mapping_rule import keep_prev_mapping_otherwise_random
+from ai.states.game_state import GameState
+from ai.STA.Strategy.strategy import Strategy
+from ai.STA.Tactic.go_to_position_pathfinder import GoToPositionPathfinder
+
+
+class LineUp(Strategy):
+    def __init__(self, game_state: GameState):
+        super().__init__(game_state)
+        self.target_position = Position(0, 0)
+        roles_to_consider = [Role.FIRST_ATTACK, Role.SECOND_ATTACK, Role.MIDDLE,
+                             Role.FIRST_DEFENCE, Role.SECOND_DEFENCE, Role.GOALKEEPER]
+
+        # self.robots_in_lineup = closest_players_to_point(self.target_position, True)
+
+        role_by_robots = [(i, self.game_state.get_player_by_role(i)) for i in roles_to_consider]
+
+        self.robots = [player for player in role_by_robots if player is not None]
+        i = 1
+        for role, player in role_by_robots:
```

Comment:
Mes commentaires précédents ont été outdated:
ligne 17 à 25 sont inutile, le `required_roles()`et `optional_roles()` handle déjà tous ça.
La for loop devrait être : 
`for i, role, player in enumerate(self.assigned_roles.items()):`